src/main/resources/.properties

```
@@ -0,0 +1,2 @@
+ReleaseDate=${project.releaseDate}
+BuildVersion=${project.version}
```

```suggestion
BuildVersion=${project.version}
```

I (and git) like trailing newlines 😄